manage.py

```diff
@@ -46,6 +46,255 @@ def db_seed():
+
+    collaboration = Value(name'cross-department collaboration')
+    db.session.add(collaboration)
+
+    open_coms = Value(name'open communication')
+    db.session.add(open_coms)
+
+    eq_iq = Value(name'eq > iq')
+    db.session.add(eq_iq)
+
+    flat_org = Value(name'flat organizations')
+    db.session.add(flat_org)
+
+    risk_taking = Value(name'risk taking > stability')
+    db.session.add(risk_taking)
+
+    hats = Value(name'wears many hats')
+    db.session.add(hats)
+                
+    heavy_team = Value(name'heavily team oriented')
+    db.session.add(heavy_team)
+
+    promotes = Value(name'promotes from within')
+    db.session.add(promotes)
+
+    mobility = Value(name'internal mobility')
+    db.session.add(mobility)
+
+    juniors = Value(name'good for juniors')
+    db.session.add(juniors)
+
+    interns = Value(name'has internships')
+    db.session.add(interns)
+
+    retention = Value(name'high employee retention')
+    db.session.add(retention)
+
+    inclusion = Value(name'actively practices inclusion')
+    db.session.add(inclusion)
+
+    work_life = Value(name'work life balance')
+    db.session.add(work_life)
+
+    personal_growth = Value(name'committed to personal growth')
+    db.session.add(personal_growth)
+
+    parents = Value(name'ideal for parents')
+    db.session.add(parents)
+
+    safe_fail = Value(name'safe environment to fail')
+    db.session.add(safe_fail)
+
+    physical_wellness = Value(name'supports physical wellness')
+    db.session.add(physical_wellness)
+    
+    psychological = Value(name'supports mental health') 
+    db.session.add(psychological)
+
+    lunch = Value(name'eats lunch together')
+    db.session.add(lunch)
+
+    flexx = Value(name'flexible work arrangements')
+    db.session.add(flexx)
+
+    light_mtgs = Value(name'light meetings')
+    db.session.add(light_mtgs)
+
+    friends = Value(name'friends outside of work')
+    db.session.add(friends)
+    
+    beer = Value(name'has good beer')
+    db.sessions.add(beer)
+
+    layout = Value(name'thoughtful office layout')
+    db.sessions.add(layout)
+
+    customer = Value(name'customer first')
+    db.sessions.add(customer)
+
+    engineer = Value(name'engineering-driven')  
+    db.sessions.add(engineer)
+                
+    product = Value(name'product-driven')
+    db.sessions.add(product)
+    
+    design_driven = Value(name'design-driven')
+    db.sessions.add(design_driven)
+
+    data_driven = Values(name'data_driven')
+    db.sessions.add(data_driven)
+
+    rapid_growth = Values(name'rapidly growing team')
+    db.sessions.add(rapid_growth)
+
+    bb = Values(name'B2B')
+    db.sessions.add(bb)
+
+    bc = Values(name'B2C')
+    db.sessions.add(bc)
+
+    self_funded = Values(name'self funded')
+    db.sessions.add(self_funded)
+                
+    tech_founders = Values(name'technical founder(s)')
+    db.sessions.add(tech_founders)
+                
+    PBC = Values(name'PBC / B-CORP')
```

Comment:
caps makes a variable a constant - I don't think it'll cause an issue, but for the sake of consistency with the rest of these seeds, this should probably be `pbc` instead of `PBC`